docs/RC/GetCapabilities/assets/GetCapabilities.txt

```diff
@@ -0,0 +1,36 @@
+participant App
+participant SDL
+participant HMI
+
+SDL<-HMI:BC.OnReady
+SDL->HMI:RC.IsReady
+SDL<--HMI:RC.IsReady (code:0)
+
+opt Request and cache capabilities on system start up
+note over SDL:<align:center>No file in `AppStorageFolder`\nfor caching capabilities data
+SDL->SDL:<align:center>Create\n`HMICapabilitiesCacheFile`
+SDL->HMI:RC.GetCapabilities
+SDL<--HMI:RC.GetCapabilities (params)
+SDL->SDL:<align:center>Persist data\nin `HMICapabilitiesCacheFile`
+App->SDL:GetCapabilities
+SDL->SDL:<align:center>Load data\nfrom `HMICapabilitiesCacheFile`
+App<--SDL:GetCapabilities (params)
+end
+
+opt Use of cached capabilities within ignition cycle
+note over SDL:<align:center>Capabilities data\nis available\nin `HMICapabilitiesCacheFile`
+App->SDL:GetCapabilities
+SDL->SDL:<align:center>Load data\nfrom `HMICapabilitiesCacheFile`
+App<--SDL:GetCapabilities (params)
+end
+
+opt Use of default capabilities data within ignition cycle
+App->SDL:GetCapabilities
+note over SDL:<align:center>Capabilities data\nis **not** available\nin `HMICapabilitiesCacheFile`
+SDL->SDL:<align:center>Load **default** capabilities\nfrom **`hmi_capabilities.json`**
+App<--SDL:GetCapabilities (params)
+SDL->HMI:RC.GetCapabilities
```

Comment:
@KhrystynaDubovyk 
The sequence was suggesed based on functional changes provided in proposal:
If capability is missing in Cache file, SDL:

1. provides App with default capabilities
2. sends request to HMI to get missing capabilities data.

> During boot up, SDL Core should check if HMICapabilitiesCacheFile file is present in AppStorageFolder from smartDeviceLink.ini
If present, SDL Core loads the contents of the file and uses that as current system HMI capabilities.
**In case any capability is missing, SDL Core should default back to hmi_capabilities.json file for corresponding capability and should send requests to get these capabilities from HMI.**

Reply:
good. it is said in the proposal. the reader is working with the guidelines. how can the reader guess it? anything in the diagram that says 

> capability is missing in Cache file?

Reply:
The information is provided in the note over SDL:

> Capabilities data is **not** available in `HMICapabilitiesCacheFile`
>
Therefore, SDL loads the default capabilities from **`hmi_capabilities.json`**
Another point to highlight the difference in the use of cached and default capabilities is provided in Opt names:

>  Use of cached capabilities within ignition cycle
> Use of default capabilities data within ignition cycle

Reply:
diagram UPD in 93fea8c38882d000c1c414790373bb5e31787c56